tests/tests/swfs/avm2/bytearray/Test.as

```diff
@@ -105,4 +105,10 @@ trace(ba1.readBoolean());
 trace(ba1.readUnsignedShort());
 trace(ba1.readShort());
 trace(ba1.readUnsignedInt());
-trace(ba1.readInt());
\ No newline at end of file
+trace(ba1.readInt());
+
+var ba2 = new ByteArray();
+ba2.position = 100;
+trace("ba2.position: " + ba2.position);
+ba2.length = 2;
+trace("ba2.position: " + ba2.length);
```

Comment:
This should be ba2.position

Reply:
Or perhaps the printed text should be `"ba2.length: "`?

Reply:
Good catch - I changed it to access `ba2.position`

Reply:
What's the point of tracing position again after only changing length? My impression was that the point of this last trace is to prove that length can be changed. But I don't mind either way.

Reply:
This PR clamps the position when `length` is changed - note that the two `ba2.position` values are different.

Reply:
Ah, alright, sorry for the trouble.